multicast/src/main/kotlin/com/dropbox/flow/multicast/Multicaster.kt

```diff
@@ -84,32 +84,43 @@ class Multicaster<T>(
     }
 
     /**
-     * The shared downstream flow. Collectors of this flow will share values dispatched by the
-     * [source] Flow.
+     * Gets a new downstream flow. Collectors of this flow will share values dispatched by a
+     * single upstream [source] Flow.
+     *
+     * @param piggybackOnly if true this downstream will cause a new upstream to start running
+     * (which only happens if no upstream is running alreay, e.g on if this is the first downstream
+     * added). Unlike a "regular" downstream, a [piggybackOnly] downstream can be closed without
```

Comment:
i don't think this is the case as we only allow piggyback downstream of piggybacking is enabled hence they'll never be closed ?